### PR TITLE
Chat

### PR DESCRIPTION
**Pull request by tsiegleauq**
Chat with contribution from @FinnStutzenstein

### CODE REVIEW

File: server/openslides/chat/models.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+
+class ChatMessage(RESTModelMixin, models.Model):
+    """"""
+
+    access_permissions = ChatMessageAccessPermissions()
+
+    objects = ChatMessageManager()
+
+    text = models.CharField(max_length=512)
```
Comment (FinnStutzenstein) [inline server/openslides/chat/models.py:75]: @tsiegleauq Please make sure, that this limit is also known in the client and do not let the user enter more chars (maybe use 500 or so). If this are too few characters, we can raise this limit.

**Comment by FinnStutzenstein** [comment]
Note for merging: https://github.com/OpenSlides/openslides3-autoupdate-service/pull/101 must be merged first and the submodule must be set to the new master afterwards.

**Review by emanuelschuetze — CHANGES_REQUESTED** [review]
I get an server error:
django.db.utils.OperationalError: no such column: chat_chatmessage.user_id
Looks that the (updated) migration could not be re-run.

How can I remove the database in my dev setup? make stop-dev does not help.

**Comment by FinnStutzenstein** [comment]
`rm server/personal_data/var/db.sqlite`

File: server/openslides/chat/models.py (hunk 1 above)
Comment (tsiegleauq) [reply server/openslides/chat/models.py:75]: server throws an error if you try. It's fine like that.

**Review by emanuelschuetze — APPROVED** [review]

**Review by FinnStutzenstein — APPROVED** [review]

**Pull request merged**